packages/core/admin/ee/server/src/services/persist-tables.ts

```diff
@@ -1,7 +1,7 @@
 import type { Core } from '@strapi/types';
 import { differenceWith, isEqual } from 'lodash/fp';
 
-interface PersistedTable {
+export interface PersistedTable {
```

Comment:
I just don't see where you are using this outside this file 🤔

Reply:
typescript was complaining about it, if another file is importing a function that references this type, it also needs to be exported to be visible (or thats what I was able to understand)